### ISSUE TITLE
BLE not active for some Govee Lights

### ISSUE THREAD

**Issue by djezlee** [Beshelmek/govee_ble_lights#27]
This is a great integration and we appreciate you for making it.

I converted several of my govee lights to this integration because I was suffering polling issues with the Http version.  However, I am unable to activate the Bluetooth on a few lights, even though I am sure they are compatible: 

- H601B
- H601A
- H6154

The only thing I haven't tried is to delete them from the Govee App and re-add them without turning on the Wifi.  Thoughts? Am I missing a crucial step?

**Comment by raresjunica** [comment]
Did you manage to connect to your H601B lights?

**Comment by djezlee** [comment]
Not at all… do you have some ideas?Best Regards,LeeSent from my iPhoneOn Dec 27, 2023, at 6:47 AM, raresjunica ***@***.***> wrote:﻿
Did you manage to connect to your H601B lights?

**Comment by raresjunica** [comment]
Didn't manage either. These are the only devices I didn't manage to connect to, but from what I see in the files, they should be supported.

**Comment by ExR90** [comment]
Reset the lights, depending on version, press the center button 4 times while holding the power button on the controller for the lights. I had to do this before they would be picked up by HA and able to add.

**Comment by janstadt** [comment]
Any luck on the H6154? Its detected and i can connect and control but its super slow and the color picker seems to take a long time to trigger different lights. Might try some other settings i guess. Using an esphome bluetooth proxy currently.

**Comment by janstadt** [comment]
Just bumping this again? I have it partially working with the H6154 via BLE but it certainly is not perfect. Anyone else have any luck on this?
